tasks/tarball.yml

```diff
@@ -17,16 +17,15 @@
 - user: name=zookeeper group=zookeeper system=yes
 
 - name: Change ownership on zookeeper directory.
-  file: path={{zookeeper_dir}} state=directory owner=zookeeper group=zookeeper recurse=yes
```

Comment:
recuse is redundant since on first run zookeeper will start and create some logs, while on impotence check will change chown files and will track changes